source/features/sort-conversations-by-update-time.tsx

```diff
@@ -57,7 +56,7 @@ function updateLink(link: HTMLAnchorElement): void {
 function init(signal: AbortSignal): void {
 	// Get issues links that don't already have a specific sorting applied
 	observe(
-		css`
+		`
```

Comment:
I can restore this when I figure out how to preserve the exact string type:

- https://github.com/fregante/code-tag/issues/12